GDJS/Runtime/runtimescene.ts

```diff
@@ -32,6 +32,10 @@ namespace gdjs {
     _requestedChange: SceneChangeRequest;
     /** Black background by default. */
     _backgroundColor: integer = 0;
+
+    /** Should the canvas be cleared before this scene rendering. */
+    clearCanvas: boolean = true;
```

Comment:
@D8H @ClementPasteau Do you think we should rather go for a setter/getter? Might allow to add/change the behavior of these in the future, while a boolean is something to maintain as a boolean forever.

Reply:
@D8H @ClementPasteau, do you have any thoughts on this?
I didn't come with the getter/setter approach because it doesn't bring any additional value here.

Reply:
Done, please check